angu-fixed-header-table.js

```diff
@@ -74,6 +79,7 @@
     function fixedHeader($timeout, $window, $q, $rootScope){
         var attr_prefix = 'fixedHeader';
         var ATTRS = {
+            TABLE_HEIGHT : 'tableHeight', // table height
```

Comment:
@rleira @ttsanev if you think that's better we add the directive prefix to it, I can add to this attribute <o/